src/charts/stackedArea/StackedAreaComponent.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import StackedAreaComponent from './StackedAreaComponent';
+import { stackedAreaData } from '../../helpers/testData';
+
+import stackedArea from './stackedAreaChart';
+
+describe('Stacked Area Chart', () => {
+
+    describe('render', () => {
+
+        it('should render the chart container', () => {
+            const wrapper = shallow(<StackedAreaComponent />);
+
+            let expected = 1;
+            let actual = wrapper.find('.stacked-area-container').length;
+
+            expect(actual).toEqual(expected);
+        });
+
+        describe('when data passed in', () => {
+            let createSpy;
+
+            beforeEach(() => {
+                createSpy = jest.spyOn(stackedArea, 'create');
+            });
+
+            afterEach(() => {
+                createSpy.mockReset();
+                createSpy.mockRestore();
+            });
+
+            it('should call the create method or the chart', () => {
+                mount(<StackedAreaComponent chart={stackedArea} data={stackedAreaData.with3Sources()} />);
+
+                let expected = 1;
+                let actual = createSpy.mock.calls.length;
+
+                expect(actual).toEqual(expected);
```

Comment:
```
expect(createSpy).toHaveBeenCalledTime(1)
```

Gives so much helpful information when it fails

Reply:
Yeah, we had something like that at first, but I think the benefits of TAP overpower it.
I guess this is something we lose in the tradeoff...